src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -78,593 +84,109 @@ const formConfig = {
-                  'ui:title': 'Lay statements or other evidence'
-                },
-                'view:evidenceTypeHelp': {
-                  'ui:description': EvidenceTypeHelp
-                }
-              }
-            }
-          },
-          schema: {
-            type: 'object',
-            properties: {
-              disabilities: {
-                type: 'array',
-                items: {
-                  type: 'object',
-                  properties: {
-                    'view:vaMedicalRecords': {
-                      type: 'boolean'
-                    },
-                    'view:privateMedicalRecords': {
-                      type: 'boolean'
-                    },
-                    'view:otherEvidence': {
-                      type: 'boolean'
-                    },
-                    'view:evidenceTypeHelp': {
-                      type: 'object',
-                      properties: {}
-                    }
-                  }
-                }
-              }
-            }
-          }
-        },
-        vaMedicalRecordsIntro: {
-          title: '',
-          path: 'supporting-evidence/:index/va-medical-records-intro',
-          showPagePerItem: true,
-          arrayPath: 'disabilities',
-          depends: (formData, index) => _.get(`disabilities.${index}.view:vaMedicalRecords`, formData),
+        expedited: {
+          title: 'Fully developed claim program',
+          path: 'additional-information/fdc',
           uiSchema: {
-            disabilities: {
-              items: {
-                'ui:title': disabilityNameTitle,
-                'ui:description': vaMedicalRecordsIntro,
-              }
-            }
-          },
-          schema: {
-            type: 'object',
-            properties: {
-              disabilities: {
-                type: 'array',
-                items: {
-                  type: 'object',
-                  properties: {}
+            'ui:description': (
+              <div>
+                <h5>Fully developed claim program</h5>
+                <p>
+                  You can apply using the Fully Developed Claim (FDC) program if
+                  you’ve uploaded all the supporting documents or supplemental
+                  forms needed to support your claim.
+                </p>
+                <a href="/pension/apply/fully-developed-claim/" target="_blank">
+                  Learn more about the FDC program
+                </a>.
+              </div>
+            ),
+            noRapidProcessing: {
+              'ui:title':
+                'Do you want to apply using the Fully Developed Claim program?',
+              'ui:widget': 'yesNo',
+              'ui:options': {
+                yesNoReverse: true,
+                labels: {
+                  Y: 'Yes, I have uploaded all my supporting documents.',
+                  N:
+                    'No, I have some extra information that I will submit to VA later.'
```

Comment:
Is it a VS Code setting to move long strings down to their own line? It only moves the whole string one character to the left, so it's not actually saving much horizontal space.